docs/Systems.md

```diff
@@ -137,6 +137,60 @@ Would you like to download the system to your cache? [Yn] Y
+The easiest way to create a new Nerves System is to check out [`nerves_system_br`](https://github.com/nerves-project/nerves_system_br) and create a configuration that contains the packages and configuration you need.
+Once you get this working and booting on your Target, you can copy the configurations and files back into a new mix project following the structure described above.
+
+Nerves systems invoke a compiler in `nerves_system` which, through an [Elixir Behaviour](http://elixir-lang.org/getting-started/typespecs-and-behaviours.html) called `Nerves.System.Provider`, calls the provider specified by the `NERVES_SYSTEM_COMPILER` and `NERVES_SYSTEM_CACHE` environment variables.
+The default behaviour is to pull these items from the HTTP cache, which uses the mirrors defined in the Nerves package configuration.
+
+If you want to force it to build locally, you will need to be on a Linux machine and configure your environment with:
+
+```
+NERVES_SYSTEM_CACHE=none
+NERVES_SYSTEM_COMPILER=local
+```
+
+There are currently a few issues when invoking Buildroot through the Mix compiler, so it may be easier to change to the `nerves_system_br` directory and run `create-build.sh` with a reference to the `defconfig` location of your new Nerves system derivative.
+
+* the `defconfig` and assets in `nerves_system_*/` are copied to the output directory at `_build/env/nerves/system/config`.
+  This will change in an upcoming release, since we are no longer going to pursue `defconfig` fragments (`system_ext`) but rather full package management.
+  This means that if you were to navigate to the output directory and go through the `make menuconfig` ... `make savedefconfig` process, the changes would actually be saved to the copied configuration located in the `_build/env/nerves/system/config` directory instead of changing the original.
+
+* Whenever the Mix compiler invokes Buildroot, it will forcefully clean the `output` directory and build from scratch.
+
+### Supporting New Target Hardware
+
+If you're trying to support a new Target, there's a bit more work involved, depending on how mature the support for that hardware is in the Buildroot community.
+
+If you can find an existing Buildroot configuration for your intended hardware:
+
+1.  Check out their full buildroot and procedure and get their stuff to boot.
+
+2.  Clone buildroot from their main repo and checkout the tag nerves is currently at:
+    https://github.com/nerves-project/nerves_system_br/blob/2d6f76406813734377bc9dad4dd5017e81ffa129/create-build.sh#L16
+
+3. Figure out where they forked off
+
+4. Compare the differences between the two to determine what standard things they changed.
+
+  * Anything added like packages and board configs can be copied into `nerves_system_br`.
```

Comment:
Packages and board configs should be in the system image. The LinkIt Smart repo is an example: https://github.com/nerves-project/nerves_system_linkit